ldaptor/ldiftree.py

```diff
@@ -289,14 +289,12 @@ def __repr__(self):
 
     def __lt__(self, other):
         if not isinstance(other, LDIFTreeEntry):
-            # We don't return NotImplemented so that we get the same
-            # result in Python2 and Python3.
-            raise (TypeError, "unorderable types: {!r} > {!r}".format(self, other))
+            return NotImplemented
```

Comment:
Maybe have a better error message here
```suggestion
            return NotImplemented('LDIFTreeEntry comparison operation not supported.')
```

Reply:
`TypeError: 'NotImplementedType' object is not callable`

Reply:
Sorry. Will  `NotImplementedError` do the job?

Reply:
NotImplemented automatically raises a              `raise (TypeError, "unorderable types: {!r} > {!r}".format(self, other)) ` style message with nice reprs etc - it's just a bit faster

Reply:
>  If all attempts return NotImplemented, the interpreter will raise an appropriate exception.